src/styles/DisplayLabyrinth.css

```diff
@@ -0,0 +1,63 @@
+.labyrinth-start {
+  padding: 50px;
+  height: 100vh;
+  background-repeat: no-repeat;
+  background-size: cover;
+  background-position: center;
+}
+
+.action-box {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  margin: 5%;
+  gap: 10px;
+  border: 5px solid black;
+  padding: 20px;
+  font-size: 20px;
+  background: linear-gradient(to top, white, beige, gray);
+}
+
+.action-button {
+  background: linear-gradient(to top, #4a6741, #374f2f, #22311d );
+  text-shadow: 0 0 10px #fff, 0 0 20px #fff, 0 0 30px #fff;
+  color: black;
+  border: 2px solid black;
+  font-weight: 600;
+  padding: 10px;
+  margin-top: 40px;
+}
+
+.action-button:hover {
+  background: linear-gradient(to top, #4a6741, #374f2f, #22311d);
```

Comment:
Since it's the same background as for the button you don't need to specify it again for the hover-state.


/Martin & Etna